docs/content/concepts/assets/asset-checks/checking-for-data-freshness.mdx

```diff
@@ -356,6 +357,39 @@ height={1944}
 
 ---
 
+## Alert on overdue assets (Dagster+ only)
+
+In Dagster+, you can set up [alerts](/dagster-plus/managing-deployments/alerts) to notify you when an asset is overdue for an update. By default, a freshness check will fail with severity `WARN` if an asset is overdue for an update. Then, in order to alert on overdue assets, you'll want to create an alert policy that triggers on asset check failures with severity `WARN` (or `ERROR` if you've overridden the default severity), for your specific assets which you've defined freshness checks for. Asset alert policies can currently be targeted at specific asset keys, asset groups, or all assets.
+
+To become more familiar with the various ways alert policies can be set up, see the guide on [managing alert policies](/dagster-plus/managing-deployments/alerts/managing-alerts-in-ui).
+
+<TabGroup>
+<TabItem name="Option 1: Use the alert policy editor UI">
+1. Navigate to the alert policy editor window in the Dagster UI.
```

Comment:
```suggestion
<TabGroup>
<TabItem name="Option 1: Dagster UI">

1. Navigate to the alert policy editor window in the Dagster UI.
```

A few things here:

- We can just say `Dagster UI` here, otherwise it's a mouthful
- A blank line is needed between `TabItem` and the content it contains, or the Markdown won't render correctly
- How do users get to the "alert policy editor window"? I don't think we call it that anywhere else, even in the UI, so this may not make sense to some users

Reply:
I added some steps to explain how to get to the editor.

Reply:
Did those changes get pushed? Everything looks the same to me atm.

Reply:
whoops you're right; I never pushed. Did so now.